tableflip/go.mod

```diff
@@ -0,0 +1,22 @@
+module tableflip
+
+go 1.17
+
+require (
+	github.com/cloudflare/tableflip v1.2.3
+	github.com/gofiber/fiber v1.14.6
```

Comment:
why do you need the old fiber version here?

Reply:
that's a mistake. I will fix it soon.